libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/CoreBot.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Dialogs.Adaptive;
+using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
+using Microsoft.Bot.Builder.Integration.Runtime.Settings;
+using Microsoft.Bot.Builder.Skills;
+using Microsoft.Bot.Schema;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Bot.Builder.Integration.Runtime
+{
+    /// <summary>
+    /// Defines the bot runtime standard implementation of <see cref="IBot"/>.
+    /// </summary>
+    internal class CoreBot : IBot
```

Comment:
The ActivityHandler implements the IBot interface, is the usage of IBot here to temporarily acknowledge that the ActivityHandler and DialogManager fulfill similar roles with respect to dispatching on Activities?

Reply:
Yes.  This was an explicit request by Eric and John to help indicate this relationship.

Reply:
Yes, along with we need to design and decide more about the relationship between ibot and dialog manager (we just had a somewhat long conversation about this so hopefully my verbal comments helped on this)_